### DISCUSSION TITLE
Kotlin Ktor and Serialization support instead of Jackson

### DISCUSSION THREAD

**Discussion by ronjunevaldoz** [ExpediaGroup/graphql-kotlin#1398] [General]
Is there any existing example for ktor and kotlin serialization implemention? I want to remove the dependency from jackson object parser instead I want to use kotlin serialization. I'm currently using https://github.com/aPureBase/KGraphQL which is a good dsl support but the owner seems not so active at all now. I want to move using ExpediaGroup/graphql-kotlin now.

**Comment by dariuszkuc** [comment]
Hello 👋 
With the recent changes (#1379) to the `graphql-kotlin-schema-generator` we no longer depend on Jackson for schema generation and data fetcher runtime (released in v `6.0.0-alpha.0`) so you can this library without a problem.

Our common server abstraction (`graphql-kotlin-server`) still depends on Jackson and we don't see it changing in foreseeable future. In order to use `kotlinx-serialization`, you would have to re-implement server logic (basically what `graphql-kotlin-server` does) to support it.
